materialchart/src/main/java/app/priceguard/materialchart/Chart.kt

```diff
@@ -97,7 +97,7 @@ class Chart @JvmOverloads constructor(
     var gridLineStrokeWidth = 6f
 
     // ZeroDp: Delete Padding. 1dp for show lines in corners and edges
-    var zeroDp = Dp(1F)
+    private var zeroDp = Dp(1F)
```

Comment:
I think `zeroDp` is not changed in any of the places. Let's change the variable to immutable.
```suggestion
    private val zeroDp = Dp(1F)
```